src/components/ExamplePage.tsx

```diff
@@ -13,7 +13,7 @@ export default function ExamplePage() {
   return (
     <>
       <Helmet>
-        <title>Hello, Plugin!</title>
+        <title data-test="example-page-title">Hello, Plugin!</title>
       </Helmet>
       <Page>
         <PageSection variant="light">
```

Comment:
Please remove empty file test-prow-e2e.sh

Reply:
> 

@spadgett I created this file which is similar to what is in console, which I think it is the script that run the test. I am trying to understand how the script works, so I left the file empty. Is this script not required for running the test in CI?

Reply:
Sorry, yeah. We'll eventually need something to kick off the tests.